template/visual-diff/integration/visualdiff.spec.js

```diff
@@ -15,7 +20,13 @@ describe('check the site for visual regressions', () => {
   pagesToCheck.forEach((route) => {
     it(`check ${route} for visual changes`, () => {
       cy.visit(route);
-      cy.eyesCheckWindow();
+      cy.eyesCheckWindow({
+        tag: route,
+        // TODO figure out why this setting fails with "left is not a number"
+        // ignore: {
+        //   selector: Cypress.env('APPLITOOLS_IGNORE_SELECTOR'),
+        // },
```

Comment:
@giltayar if I enable this setting the build fails with this error:

```
  1) check the site for visual regressions
       "after all" hook for "check /about for visual changes":
     Error: Eyes-Cypress detected diffs or errors during execution of visual tests:
       Passed - 0 tests
       Diffs detected - 0 tests
       Errors - 2 tests
         ✖ [Eyes test not started] : Error: IllegalType: left is not a number
         ✖ [Eyes test not started] : Error: IllegalType: left is not a number
```

I was unable to find any information about this error, so I'm not sure what's going on here

Reply:
Yeah, bad validation. My guess is that our code sees the `ignore` object, but the selector is undefined, and so it's confused in thinking that it's a "region" selector (with `top`, `left`, ...) coordinates and not a "selector" region. Hence the `left` is not a number (because it's `undefined`).

I would use the fact that `ignore` can be an array and do something like this:

```
ignore: Cypress.env('APPLITOOLS_IGNORE_SELECTOR') ? [{selector: Cypress.env('APPLITOOLS_IGNORE_SELECTOR')}] : []
```

Reply:
I refactored this to turn a list of selectors (e.g. `.one,.two`) into an array of arguments:
```js
[
  { selector: '.one' },
  { selector: '.two' },
]
```